<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="rcu_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>RCU Status</string>
    </property>
    <widget class="QLabel" name="glv_voltage_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GLV Battery Voltage:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="glv_voltage">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="rcu_temp_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RCU Temperature:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="rcu_temperature">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="bms_voltage_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>BMS Voltages</string>
    </property>
    <widget class="QLabel" name="avg_voltage_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Average:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="avg_voltage">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="low_voltage_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Low:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="low_voltage">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="high_voltage_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>High:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="high_voltage">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="total_voltage_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Pack Voltage:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="total_voltage">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>279</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
